assets/app/views/_overview-deployment.html

```diff
@@ -78,7 +78,7 @@
     </div>
 
     <!-- Pods donut chart and scaling controls -->
-    <div column>
+    <div column class="overview-pods">
```

Comment:
I feel like the chart and the scaling controls are so closely related it should really be one directive. The scaling controls adjust the values in the chart and the chart (should) react immediately.